RealismOverhaul.netkan

```diff
@@ -56,15 +55,10 @@
         { "name" : "ForgottenRealEngines" },
         { "name" : "KWRocketryRedux" },
         { "name" : "FilterExtensions" },
-        { "name" : "LazTekSpaceXExploration" },
-        { "name" : "LazTekSpaceXHistoric" },
-        { "name" : "LazTekSpaceXLaunch" },
         { "name" : "NearFutureConstruction" },
-        { "name" : "NearFutureElectric" },
         { "name" : "NearFuturePropulsion" },
         { "name" : "NearFutureSolar" },
         { "name" : "NearFutureSpacecraft" },
-        { "name" : "PorkjetHabitats" },
         { "name" : "RLAContinued" },
```

Comment:
Use continued mod `RLARecontinued` for 1.3.1

Reply:
changed